src/hotspot/share/runtime/thread.cpp

```diff
@@ -457,11 +455,6 @@ Thread::~Thread() {
   delete handle_area();
   delete metadata_handles();
 
-  // SR_handler uses this as a termination indicator -
```

Comment:
As noted previously we need a replacement for this as a proxy for a Thread (not JavaThread) terminating.

Reply:
Fixed